hetio/neo4j.py

```diff
@@ -28,18 +42,32 @@ def export_neo4j(graph, uri, node_queue=100, edge_queue=100):
         if 'name' not in db_graph.schema.get_indexes(label):
             db_graph.schema.create_index(label, 'name')
 
+
     # Create nodes
     creator = Creator(db_graph, node_queue)
-    for node in graph.get_nodes():
+
+    queue = graph.get_nodes()
+    if show_progress:
+        total_nodes = hetio.stats.get_metanode_df(graph)["nodes"].sum()
+        queue = tqdm(queue, total = total_nodes, desc = "Importing nodes")
+
+    for node in queue:
         label = as_label(node.metanode)
         data = sanitize_data(node.data)
         neo_node = py2neo.Node(label, identifier=node.identifier, name=node.name, **data)
         creator.append(neo_node)
     creator.create()
 
+
     # Create edges
     creator = Creator(db_graph, edge_queue)
-    for edge in graph.get_edges(exclude_inverts=True):
+
+    queue = graph.get_edges(exclude_inverts=True)
+    if show_progress:
+        total_edges = hetio.stats.get_metaedge_df(graph)["edges"].sum()
```

Comment:
In https://github.com/dhimmel/hetio/commit/903a4c8209925605a7b23556d1d889514b8840dd, I added `n_nodes` and `n_edges` as graph attributes. Therefore, you should be able to do `total_edges = graph.n_edges` (and similar for nodes).

You would have to fetch the upstream changes and rebase your branch on them. Then force push. Do you know how to do that?